plugins/factoids.py

```diff
@@ -349,18 +350,17 @@ async def tag_info(self, ctx: Context, *, name: str) -> None:
             stmt = select(Alias).where(Alias.id == alias.id).order_by(Alias.uses.desc())
             aliases = (await session.execute(stmt)).scalars()
 
-            created_at = int(alias.factoid.created_at.replace(tzinfo=timezone.utc).timestamp())
-            used_at = None
-            if alias.factoid.used_at is not None:
-                used_at = int(alias.factoid.used_at.replace(tzinfo=timezone.utc).timestamp())
+            used_at = alias.factoid.used_at
+            used_at_str = format(", last on {!f} ({!R})", used_at, used_at) if used_at else ""
```

Comment:
perhaps an unnecessary refactor - this code could be substituted back in to 363, but given the new formatting this would be messy

Reply:
`if used_at is not None`